test/fuzz/TrancheWrapper/PreviewWithdraw.fuzz.t.sol

```diff
@@ -0,0 +1,66 @@
+/**
+ * Created by Pragma Labs
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+pragma solidity 0.8.22;
+
+import { TrancheWrapper_Fuzz_Test } from "./_TrancheWrapper.fuzz.t.sol";
+
+import { stdStorage, StdStorage } from "../../../lib/accounts-v2/lib/forge-std/src/StdStorage.sol";
+import { TrancheExtension } from "../../utils/extensions/TrancheExtension.sol";
+
+/**
+ * @notice Fuzz tests for the function "totalAssets" of contract "Tranche".
+ */
+contract PreviewWithdraw_TrancheWrapper_Fuzz_Test is TrancheWrapper_Fuzz_Test {
+    using stdStorage for StdStorage;
+    /* ///////////////////////////////////////////////////////////////
+                              SETUP
+    /////////////////////////////////////////////////////////////// */
+
+    function setUp() public override {
+        TrancheWrapper_Fuzz_Test.setUp();
+    }
+
+    function testFuzz_Success_previewWithdraw_NonZeroSupply(
+        uint80 vas,
+        uint256 totalSupply,
+        uint128 totalAssets,
+        uint256 assets
+    ) public {
+        // Given: totalSupply is bigger than zero.
+        // And: no overflows.
+        totalSupply = bound(totalSupply, 1, type(uint256).max - vas);
+        assets = bound(assets, 0, type(uint256).max / (totalSupply + vas));
+
+        // And: totalAssets() + VAS is non-zero.
+        vm.assume(uint256(totalAssets) + vas > 0);
+
+        // And: Tranche state is set.
+        tranche = TrancheExtension(setTrancheState(vas, totalSupply, totalAssets));
```

Comment:
same as above - calls should be done on `trancheWrapper `instead of `tranche`. As we are deploying a new tranche here, we should deploy a new trancheWrapper for that tranche and run tests on that trancheWrapper